api/services/reviewService.js

```diff
@@ -0,0 +1,55 @@
+const jwt = require("jsonwebtoken");
+
+const { reviewDao } = require("../models");
+const { commentValidate } = require("../utils/validate-check");
+
+const createReview = async (accessToken, rating, comment, movieId) => {
+  await commentValidate(comment);
+
+  const userId = jwt.verify(accessToken, process.env.JWT_SECRET).id;
+  const checkUserMovie = await reviewDao.checkUserMovie(userId, movieId);
+  if (checkUserMovie == false) {
```

Comment:
```suggestion
  if (!checkUserMovie) {
```
부정연산자를 붙여서 쓰는것이 가독성이 훨씬 좋고, 더 일반적으로 많이 쓰입니다.